.gitlab-ci.yml

```diff
@@ -5,6 +5,28 @@
   tags:
     - docker
 
+.with-compilation-database:
```

Comment:
This compilation database has a little disadvantage: it limits the cppcheck to the "linux" headers and sources. So only the dummy bwl and bpl are checked.

This can be fixed later by adding different variants (but that's more work because we need stubs for the different libraries that are used by bwl/bpl).

Reply:
In that case, maybe it would be better to keep using the hardcoded list of includes in the meantime? What do you think?

Reply:
I dropped it and used the static list of includes for now.